ambry-clustermap/src/test/java/com.github.ambry.clustermap/TestUtils.java

```diff
@@ -39,20 +39,24 @@
   static final int DEFAULT_XID = 64;
 
   enum ReplicaStateType {
+    // @formatter:off
     SealedState,
     StoppedState
+    // @formatter:om
   }
 
   /**
    * Resource state associated with datanode, disk and replica.
    */
   enum ResourceState {
+    // @formatter:off
     Node_Up,
```

Comment:
No need to remove these, but for some reason, my formatter keeps the enum like this without compressing it onto one line like it used to. It may be different on different versions of LI intellij.